torch/csrc/autograd/profiler.h

```diff
@@ -123,7 +123,7 @@ struct Event final {
     return device_;
   }
 private:
-  int64_t cpu_ns_; // signed to allow for negative intervals
+  int64_t cpu_ns_ = 0; // signed to allow for negative intervals
```

Comment:
That again is wasteful. It's always initialized *in the constructor*, so you'll literally write a `0` to it just to overwrite it with sth else.

Reply:
Well, no, it's not being initialized in any  constructor. This clang tidy check is pretty legit about this

Reply:
It is initialized in all constructors, but indirectly via the `record` method (which exists *purely* because we don't want to duplicate its code).